docs/sources/troubleshooting/support-bundles/index.md

```diff
@@ -53,3 +53,64 @@ To generate a support bundle and send the support bundle to Grafana Labs via a s
    Grafana downloads the support bundle to an archive (tar.gz) file.
 
 1. Attach the archive (tar.gz) file to a support ticket that you send to Grafana Labs Technical Support.
+
+## Support bundle configuration
+
+You can configure the following settings for support bundles:
+
+```ini
+# Enable support bundle creation (default: true)
+enabled = true
+# Only server admins can generate and view support bundles (default: true)
+server_admin_only = true
+# If set, bundles will be encrypted with the provided public keys separated by whitespace
+public_keys = ""
+```
+
+### Support bundle encryption
+
+You can encrypt support bundles with an [age](age-encryption.org) formatted public key.
+
+Ensure [age](https://github.com/FiloSottile/age#installation) is installed on your system.
+
+#### Generating a key pair
+
+```bash
+$ age-keygen -o key.txt
+Public key: age1ql3z7hjy54pw3hyww5ayyfg7zqgvc7w3j2elw8zmrj2kg5sfn9aqmcac8p
+```
+
```

Comment:
nit: I think there could be a new header here to separate key generation, encryption and decryption - something like `#### Encrypting a support bundle`.